source/modules/models/fileStore.js

```diff
@@ -384,6 +384,56 @@ class FileStore {
 
         return true;
     }
+
+    /**
+     * Sort the file entries in the file tree so that directories are listed before text files, and sort in natural order, should be called on render. 
+     */
+    sort_files() {
+        const naturalCompare = (a, b) => {
+            const ax = [], bx = [];
+
+            a.replace(/(\d+)|(\D+)/g, (_, $1, $2) => {
```

Comment:
Can you add comment on the purpose of these lines.

Reply:
Added comments :)

Reply:
👍